Simple Search Engine/task/src/search/ISearchMethod.java

```diff
@@ -0,0 +1,7 @@
+package search;
+
+import java.util.ArrayList;
+
+public interface ISearchMethod {
+    ArrayList<String> searchFor(String[] queryParts);
```

Comment:
Weet niet of je al van de JavaDoc bent?? Anders gaan toevoegen, dat helpt een ander met het begrijpen van de code en waarom bepaalde keuzes zijn gemaakt

Reply:
Done, op sommige punten.